R/merge.R

```diff
@@ -85,3 +53,35 @@ shift_output <- function(recs) {
   }
   do.call(rbind, recs)
 }
+
+#' @rdname merge_casts
+#' @export
+
+clear_screen <- function() {
+  output <- tibble::tibble(
+    time = 0,
+    type = "o",
+    data = "\u001b[H\u001b[2J"
+  )
+  new_merge_cast_command("clear_screen", output = output)
+}
+
+#' @rdname merge_casts
+#' @param secs Number of seconds to wait.
+#' @export
+
+pause <- function(secs) {
+  output <- tibble::tibble(
+    time = as.numeric(secs),
+    type = "o",
+    data = ""
+  )
+  new_merge_cast_command("pause", output = output)
+}
+
+new_merge_cast_command <- function(command, output) {
+  structure(
+    list(command = command, output = list(output = output)),
+    class = "merge_cast_command"
+  )
+}
```

Comment:
Why rewrite all these?

Reply:
To have`merge_casts()` appear first.

Reply:
Why is that important?

Reply:
The order here looks awkward to me.

![image](https://github.com/user-attachments/assets/95641c99-36fd-480c-bfbd-62b62be55880)